### PR TITLE
New Devices (Matter Switch) LetainPai Smart Switch(s)

### PR DESCRIPTION
**Pull request by lelandblue**
Check all that apply

# Type of Change

- [ X] WWST Certification Request
- [ ] Bug fix
- [ ] New feature
- [ ] Refactor

# Checklist

- [ X ] I have performed a self-review of my code
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have verified my changes by testing with a device or have communicated a plan for testing
- [ ] I am adding new behavior, such as adding a sub-driver, and have added and run new unit tests to cover the new behavior

# Description of Change
Adding fingerprints for Three CN Certification Request for Matter Switch Devices. All devices are in the DCL with a device type of x103. 
LeTianPai Smart Switch(with Neutral) 3 Gangs
LeTianPai Smart Switch(with Neutral) 2 Gangs
LeTianPai Smart Switch(with Neutral) 1 Gang

I will also reach out regarding translations of this device, but I believe this can move forward.

# Summary of Completed Tests
I believe this will be further tested in CN.

### CODE REVIEW

File: drivers/SmartThings/matter-switch/fingerprints.yml [hunk 1]
```diff
@@ -218,6 +218,22 @@ matterManufacturer:
     vendorId: 0x1021
     productId: 0x0005
     deviceProfileName: switch-binary
+#LeTainPai
+  - id: "5163/4100"
+    deviceLabel: LeTianPai Smart Switch(with Neutral) 3 Gangs
```
[the remark this answers is not in the document]
Comment (Detailzc) [reply drivers/SmartThings/matter-switch/fingerprints.yml:223]: Hi,for 1 / 2 / 3 switch，devicelabel should to match 
LeTianPai Smart Switch 1 Way
LeTianPai Smart Switch 2 Way
LeTianPai Smart Switch 3 Way

[the remark this answers is not in the document]
Comment (Detailzc) [reply drivers/SmartThings/matter-switch/fingerprints.yml:223]: > The ProductName in the DCL is "LeTianPai Smart Switch 3 Way" - does the deviceLabel here need to match?

Hi,for 1 / 2 / 3 switch，devicelabel should to match 
LeTianPai Smart Switch 1 Way
LeTianPai Smart Switch 2 Way
LeTianPai Smart Switch 3 Way

[the remark this answers is not in the document]
Comment (lelandblue) [reply drivers/SmartThings/matter-switch/fingerprints.yml]: Thank you @Detailzc and @nickolas-deboom for your feedback. I have updated the PR with the provided device labels in this thread.

**Review by nickolas-deboom — APPROVED** [review]

**Review by hcarter-775 — APPROVED** [review]

File: drivers/SmartThings/matter-switch/fingerprints.yml [hunk 2]
```diff
@@ -218,6 +218,22 @@ matterManufacturer:
     vendorId: 0x1021
     productId: 0x0005
     deviceProfileName: switch-binary
+#LeTainPai
```
Comment (Detailzc) [inline drivers/SmartThings/matter-switch/fingerprints.yml:221]: Hi @lelandblue ,this brand name is LeTianPai,not LeTainPai,please revise it

**Pull request merged**